File: generated-sources/scala-finch/mojang-api/src/main/scala/com/github/asyncmc/mojang/api/scala/finch/model/SecurityAnswerId.scala
package com.github.asyncmc.mojang.api.scala.finch.model

import io.circe._
import io.finch.circe._
import io.circe.generic.semiauto._
import io.circe.java8.time._
import org.openapitools._

/**
 * A security question answer id given by Mojang
 * @param id A number giver by Mojang to reply a question
 */
case class SecurityAnswerId(id: Int
                )

object SecurityAnswerId {
    /**
     * Creates the codec for converting SecurityAnswerId from and to JSON.
     */
    implicit val decoder: Decoder[SecurityAnswerId] = deriveDecoder
    implicit val encoder: ObjectEncoder[SecurityAnswerId] = deriveEncoder
}

File: generated-sources/dart-jaguar/mojang-api/lib/model/security_answer_id.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

part 'security_answer_id.jser.dart';

class SecurityAnswerId {
   /* A number giver by Mojang to reply a question */
  @Alias('id')
  final int id;
  

  SecurityAnswerId(
    

{
    
     this.id = null 
    }
  );

  @override
  String toString() {
    return 'SecurityAnswerId[id=$id, ]';
  }
}

@GenSerializer()
class SecurityAnswerIdSerializer extends Serializer<SecurityAnswerId> with _$SecurityAnswerIdSerializer {

}
